controller/handler/handler.go

```diff
@@ -3,50 +3,54 @@ package handler
 import (
 	"net/http"
 
-	"github.com/TwiN/gatus/v3/config/ui"
-	"github.com/TwiN/gatus/v3/security"
+	"github.com/TwiN/gatus/v3/config"
 	"github.com/TwiN/health"
 	"github.com/gorilla/mux"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-func CreateRouter(staticFolder string, securityConfig *security.Config, uiConfig *ui.Config, enabledMetrics bool) *mux.Router {
+func CreateRouter(staticFolder string, config *config.Config) *mux.Router {
 	router := mux.NewRouter()
-	if enabledMetrics {
-		router.Handle("/metrics", promhttp.Handler()).Methods("GET")
-	}
 	api := router.PathPrefix("/api").Subrouter()
 	protected := api.PathPrefix("/").Subrouter()
 	unprotected := api.PathPrefix("/").Subrouter()
-	if securityConfig != nil {
-		if err := securityConfig.RegisterHandlers(router); err != nil {
-			panic(err)
+	if config != nil {
```

Comment:
The config is validated before `CreateRouter` is called, so it shouldn't be necessary to check if the config is nil.

Reply:
Resolved by #307